examples/ruby/spec/interactions/frames_spec.rb

```diff
@@ -3,5 +3,40 @@
 require 'spec_helper'
 
 RSpec.describe 'Frames' do
+  #set session
   let(:driver) { start_session }
+  let(:wait) { Selenium::WebDriver::Wait.new(timeout: 2) }
+
+  it 'interacts with elements inside iframes' do
+    #navigate to web page
+    driver.navigate.to 'https://www.selenium.dev/selenium/web/iframes.html'
+    # Switch to iframe using WebElement
+    iframe = driver.find_element(id: 'iframe1')
+    driver.switch_to.frame(iframe)
+    expect(driver.page_source.include?('We Leave From Here')).to be true
```

Comment:
```suggestion
    expect(driver.page_source).to include('We Leave From Here')
```

Reason. RSpec exposes these booleans and it's a way of getting a better error message if it occurs